src/molecules/BottomSheet/BottomSheetSectionList.native.js

```diff
@@ -0,0 +1,10 @@
+import { SectionList } from 'react-native';
```

Comment:
```suggestion
import { SectionList as BottomSheetSectionList } from 'react-native';
BottomSheetSectionList.displayName = 'BottomSheetSectionList';
```

and then you don't need anything else I guess